_scripts/main.js

```diff
@@ -1,7 +1,47 @@
 import Nav from './modules/nav';
 import Homepage from './modules/homepage';
+import Typed from 'typed.js';
+// import Homepage from './modules/about';
```

Comment:
when you import you need to call it something different import About from './modules/about